lib/qwerty/crop.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'draw'
+module Qwerty
+  class Draw
+    class Crop < Ruote::Participant
+      def on_workitem
+        workitem.fields['draw']['crop'] = {
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.